apps/researcher/src/components/language-selector.tsx

```diff
@@ -0,0 +1,99 @@
+import {CheckIcon, ChevronUpDownIcon} from '@heroicons/react/20/solid';
+import {Combobox} from '@headlessui/react';
+import {useState} from 'react';
+import classNames from 'classnames';
+import ISO6391 from 'iso-639-1-dir';
+import {LanguageCode} from 'iso-639-1-dir/dist/data';
+
+interface Props {
+  value: string;
+  setValue: (value: string) => void;
+}
+
+export default function LanguageSelector({value, setValue}: Props) {
+  const [query, setQuery] = useState('');
+
+  const filteredLocales: LanguageCode[] = query
```

Comment:
Are `filteredLocale` and its type `LanguageCode` two different names for the same thing? If so, use one name instead (e.g. `filteredLanguageCodes)?